From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Mon, 10 Jul 2023 22:05:08 +0200
Subject: [PATCH] Material fixes - Add missing value to isFuel

Adds the missing legacy value `LEGACY_DEAD_BUSH` to `Material.isFuel`.

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/org/bukkit/Material.java b/src/main/java/org/bukkit/Material.java
index 27368dc623859eade5a01abe3033a5ff225e7f79..a64e28e4ccf7a2357087f82d6b00a77561f8abc3 100644
--- a/src/main/java/org/bukkit/Material.java
+++ b/src/main/java/org/bukkit/Material.java
@@ -8322,6 +8322,7 @@ public enum Material implements Keyed, Translatable, net.kyori.adventure.transla
             case LEGACY_WOOD_STEP:
             case LEGACY_SAPLING:
             case LEGACY_STICK:
+            case LEGACY_DEAD_BUSH: // Fiddle - add missing Material values that are fuel
             case LEGACY_WOOD_BUTTON:
             case LEGACY_WOOL:
             case LEGACY_CARPET:
